src/components/loginPage.jsx

```diff
@@ -1,6 +1,7 @@
 import axios from "axios";
 import React from "react";
 import { useState } from "react";
+import { NotificationContainer, NotificationManager } from "react-notifications";
```

Comment:
You don't need to import the NotificationContainer component to this page. consider removing it